varia/functions.php

```diff
@@ -225,17 +225,7 @@ function varia_setup() {
 				'enable_theme_default' => true,
 			)
 		);
-		
-		// Add support for Content Options.
-		add_theme_support( 'jetpack-content-options', array(
-			'blog-display' => 'content',
-			'featured-images' => array(
-				'archive'         => true,
-				'archive-default' => true,
-				'post'            => true,
-				'page'            => true,
```

Comment:
The problem with this is that I'm not sure whether `get_theme_mod( 'show_featured_image_on_pages', false )` is saved, but if it is, would it not be possible to add `page-default` based on that setting here so that users are still able to access the benefits of Content Options?

Reply:
What else do the content options provide?  I'm not familiar with what else this enables for the theme.

Reply:
Previously, people were only able to hide Featured Images for pages. This led to requests of people asking to be able to hide Featured Images for posts and on their home page, as possible on other themes. Content Options gives users this ability (see the issues referenced in #7518!).

Reply:
It does default to "false" [in the varia implementation](https://github.com/Automattic/themes/pull/7566/files#diff-331e24fa45506562fc326a2c7f7e706b85795bc01b34ed86bf2539edca8e4a30R501).

So we would need to start there.  And also the same for all of the other options; we can't turn on displaying the featured image without user intervention.

Also if the user had already set it to 'true' then we would need to send that through to the jetpack-content-options, so just defaulting to 'false' wouldn't be enough, that value would also have to include whatever value was stored in the customizer.

It seems overly complex to leverage Jetpack to manage the display of the featured image.

Reply:
> So we would need to start there. And also the same for all of the other options; we can't turn on displaying the featured image without user intervention.

This is what this PR would end up doing though. :) The users would have their Featured Images displayed without any control/consent over this. 

> Also if the user had already set it to 'true' then we would need to send that through to the jetpack-content-options, so just defaulting to 'false' wouldn't be enough, that value would also have to include whatever value was stored in the customizer.

Please forgive me if I'm misunderstanding here, but if that value is no longer stored, I don't see how this resolves Automattic/wp-calypso#85369? I think that the error which I made was not setting `page-default` to false. 

> It seems overly complex to leverage Jetpack to manage the display of the featured image.

That is fair enough, but I'd argue that this PR should implement all the options properly within Varia rather than just reverting the change (it still labels it as "Jetpack Content Options", after all!).